Faca e Cutelo de Ouro para Chefs Refinados (Bounty)

VictorJob opened the pull request:
<!-- Você pode usar as guidelines dos space-wizards: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- NOTE: All code submitted to this repository is ALWAYS licensed under the AGPL-3.0-or-later license. 
The REUSE Specification headers or separate .license files indicate a secondary license (e.g., MPL or MIT) solely to facilitate 
integration for projects that do not use the AGPL license. This secondary license does not replace the fact that AGPL-3.0-or-later remains the primary and binding license. 
Uncomment and modify the following line if you wish to change the license from the default of AGPL.-->
<!--- LICENSE: AGPL -->

## Sobre a PR
<!-- O que ela muda? -->
 Esse PR adiciona adiciona uma faca e cutelo de ouro para chefs com mais de 100 horas no cargo.


## Anexos
<!-- Imagens ou videos sobre as mudanças da PR (opcional | recomendado). -->
<img width="265" height="218" alt="image" src="https://github.com/user-attachments/assets/3338138c-d5f9-4edb-bff8-6336fe517af5" />

## Requirimentos
<!--Confirme botando X entre os colchetes [X]: -->
- [X] Eu li e estou seguindo a [Politica de pull requests e changelog](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] Eu adicionei anexos à esta PR ou não precisa de imagens in-game.

**Changelog**
<!-- Adicione uma entrada para changelog do servidor, bote o emoji :cl: e siga com as mudanças assim como o exemplo.
:cl: em branco fara com que seu nome na changelog apareça como do github, adicione um nome depois do emoji para mudar seu nome na changelog. -->

:cl: VictorJob
- add: Cutelo e Faca de ouro para chefs




<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Added two gold-themed chef melee weapons (Gold Kitchen Knife, Gold Butcher Cleaver).
  * New Chef trinkets loadout group allowing Chefs to pick one trinket loadout.
  * New Chef loadouts: Gold Kitchen Knife or Gold Butcher Cleaver, gated to veteran Chefs (100 hours).

* **Style**
  * Added asset metadata and sprites for the new items.

* **Localization**
  * Added pt-BR names and descriptions for both weapons.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

VictorJob committed:
adiciona novas sprites douradas de faca e cutelo de cozinha

VictorJob committed:
corrige o nome da pasta /Gabystation/.../Melee (estava Meele antes)

VictorJob committed:
adiciona faca de cozinha e cutelo do açougueiro de ouro

VictorJob committed:
corrige nome de outra pasta de novo...

GabyChangelog committed:
chore: Automatically update REUSE headers

GabyChangelog committed:
chore: Automatically update REUSE headers

AgentePanela commented on Resources/Prototypes/_Gabystation/Loadouts/loadout_groups.yml:
```diff
@@ -29,3 +30,12 @@
+  id: TrinketsChef
+  name: loadout-group-trinkets-job
+  minLimit: 0
+  maxLimit: 2
```
```suggestion
  maxLimit: 1
```
Acho mais legal eles poderem escolher entre um ou outro

AgentePanela commented on Resources/Prototypes/_Gabystation/Entities/Objects/Weapons/Melee/knifes.yml:
```diff
@@ -0,0 +1,63 @@
+  name: Faca de Cozinha de Ouro
+  parent: BaseKnife
+  id: GoldKitchenKnife
+  description: Banhada a ouro apenas para os chefs mais refinados. Uma faca de chef de uso geral, fabricada pela Asters Merchant Guild.
```
(nao to pedindo pra voce mudar) mas eu particularmente prefiro escrever as coisas em ingles e dai traduzir no ftl pro caso de algum outro codigo quiser portar algum dia, mas dai é de voce

Kyoth25f commented on Resources/Prototypes/_Gabystation/Entities/Objects/Weapons/Melee/knifes.yml:
```diff
@@ -0,0 +1,63 @@
+
+- type: entity
+  name: Cutelo de Açougueiro de Ouro
+  parent: BaseKnife
```
mais simples usar `ButchCleaver` como parent e só redefinir o sprite

Kyoth25f commented on Resources/Prototypes/_Gabystation/Entities/Objects/Weapons/Melee/knifes.yml:
```diff
@@ -0,0 +1,63 @@
+
+- type: entity
+  name: Faca de Cozinha de Ouro
+  parent: BaseKnife
```
mais simples usar `KitchenKnife` como parent e só redefinir o sprite

Kyoth25f committed:
tweak: Aplica sugestão da tradução

Kyoth25f committed:
tweak: Reparenta

Caso atualizem uma das facas no goob, essa atualização vai contemplar as nossas também.
